packages/neutrino/bin/start.js

```diff
@@ -1,31 +1,57 @@
 const { start } = require('../src');
 const ora = require('ora');
+const dns = require('dns');
+const os = require('os');
+
+const platformHostName = os.hostname();
+const whenIPReady = new Promise((done, failed) => {
```

Comment:
Switch to [futures](https://github.com/fluture-js/Fluture) instead of Promises:

```js
const Future = require('fluture');

const whenIpReady = Future.node(done => lookup(host, done));
```